library/src/main/java/com/okta/oidc/clients/sessions/SessionClientImpl.java

```diff
@@ -58,7 +60,7 @@ public void getUserProfile(final RequestCallback<UserInfo, AuthorizationExceptio
 
     public void introspectToken(String token, String tokenType,
                                 final RequestCallback<IntrospectInfo, AuthorizationException> cb) {
-        mDispatcher.submit(() -> {
+        mFutureTask = mDispatcher.submit(() -> {
```

Comment:
What the user will get if he is going to have 2 calls for user profile in 2 different places of the app ?

Reply:
will have to cancel the current task.